guides/gatherers.md

```diff
@@ -1575,3 +1577,119 @@ Example output (in Rhai):
 // hacluster_has_default_password
 true;
 ```
+
+<span id="ini_files"></span>
+
+### ini_files@v1
+
+**Argument required**: yes.
+
+This gatherer fetches the content from a configuration file in INI format. The configuration file is specified as argument, choosen from a list of allowed files.
+Currently whitelisted files are:
+
+- `global.ini`
+
+Each fact request can return one or more item, one for each found file; multiple files can occur when the host has configured more than one SAP system. Each item then has a `sid` field with the system id and a `value` field with the actual content of the file.
```

Comment:
is it only one item or multiple items? 

Another question why do you want to use the `;`
Maybe splitting into two sentences would make sense?

Reply:
I did not understand the question. For every fact request, the response is always a list. The list has one item for every found file, according to the current host status.

Some hosts have more than a SAP installation, thus more than a global.ini file. The agent iterates through such files and returns the content as list items.